pkg/cloud/rgraph/rnode/tcproute/type_trait.go

```diff
@@ -43,5 +43,6 @@ func (*tcpRouteTypeTrait) FieldTraits(meta.Version) *api.FieldTraits {
 	dt.AllowZeroValue(api.Path{}.Pointer().Field("Rules").AnySliceIndex().Pointer().Field("Action").Pointer().Field("Destinations"))
 	dt.AllowZeroValue(api.Path{}.Pointer().Field("Rules").AnySliceIndex().Pointer().Field("Action").Pointer().Field("OriginalDestination"))
 
+	dt.System(api.Path{}.Pointer().Field("Name"))
```

Comment:
please clean up commits, you should end up with 2 commits - the fix for name issue and e2e test for tcproute. Please include detail description of the issue in PR comments

Reply:
And also add comment why this field is system. Do we want to do this for all types or tcproute only?

Reply:
Done.